src/main/java/randoop/main/GenTests.java

```diff
@@ -537,6 +597,34 @@ public boolean handle(String[] args) {
     return true;
   }
 
+  /**
+   * Tallies the number of sequences an operation occurs in.
+   *
+   * @param tallyMap the count map to increment for each operation
+   * @param sequences sequences to process for operations
+   */
+  private void tallyOperationsInSequences(
+      Map<TypedOperation, Integer> tallyMap, List<ExecutableSequence> sequences) {
+    for (ExecutableSequence es : sequences) {
+      HashSet<TypedOperation> ops = new HashSet<>();
+      SimpleList<Statement> statements = es.sequence.statements;
+      for (int i = 0; i < statements.size(); i++) {
```

Comment:
Why is the ops set needed?
Maybe it's to ensure that each operation contributes only once to the result.  That is, maybe the result is the number of tests that the operation appears in, not the number of occurrences of the operation (as I had assumed).
The specification is unclear on what is being computed.  Could you please document that?  And document the `testOccurrences` and `flakyOccurrences` variables too?